pkg/volume/nfs/nfs.go

```diff
@@ -66,9 +67,21 @@ func (plugin *nfsPlugin) Name() string {
 	return nfsPluginName
 }
 
+func findMount() bool {
+	exe := exec.New()
+	cmd1 := exe.Command("/bin/ls", "/sbin/mount.nfs")
+	_, err1 := cmd1.CombinedOutput()
+	cmd2 := exe.Command("/bin/ls", "/sbin/mount.nfs4")
+	_, err2 := cmd2.CombinedOutput()
+	return (err1 == nil || err2 == nil)
+}
+
```

Comment:
Do you think we should make conditional compilation files so that a linux check can differ from other platforms?